pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-    "mkdocstrings>=0.19",
+    "mkdocstrings>=0.26",
```

Comment:
Not sure if this is necessary. Changed this since `mkdocstrings` 0.26.0 starts requiring autorefs v1+.

Reply:
Let's depend on mkdocs-autorefs v1 directly then.

Reply:
Actually, the `autoref` element was introduced in v1.1. I've pushed to your branch :slightly_smiling_face: